src/main/java/org/templegalaxia/model/Ring.java

```diff
@@ -0,0 +1,29 @@
+package org.templegalaxia.model;
+
+import heronarts.lx.model.LXAbstractFixture;
+import heronarts.lx.model.LXPoint;
+import java.util.HashMap;
+import java.util.Map;
+
+public class Ring extends LXAbstractFixture {
```

Comment:
if Ring extends `LXModel` we'll get all of the features the LX model does, mainly model.xmin, model.radius, etc etc. Opinions?